File: nodejs/mqtt-client.js
var mqtt=require('mqtt');
var client  = mqtt.connect('mqtt://mqtt.bconimg.com');
 
client.on('connect', function () {
  client.subscribe('/beacons');
});
 
client.on('message', function (topic, message) {
     // message is Buffer 
      console.log(message.toString());
});

File: ruby/mqtt-client.rb
require 'rubygems'
require 'mqtt'


#client=MQTT::Client.new
#client.host='mqtt.bconimg.com'
#client.connect()
#client.subscribe('/beacons')
#
#client.get do |topic,message|
#	puts message;
#end


MQTT::Client.connect('mqtt.bconimg.com') do |c|
     c.get('/beacons') do |topic,message|
         puts "#{topic}: #{message}"
     end
end
